chainconsumer/analysis.py

```diff
@@ -5,6 +5,7 @@
 from scipy.ndimage.filters import gaussian_filter
 from .helpers import get_smoothed_bins, get_grid_bins, get_latex_table_frame
 from .kde import MegKDE
+from fastkde import fastKDE
```

Comment:
Given fastkde has cython as a dependency, which I know some people have issues installing, we should make fastkde a conditional import so it only happens if requested. To stop this happening in multiple locations in code, it might be good to add a method `perform_kde` which is just imports fastkde, does the kde, and returns the pdf and axes. Happy to do that when Im back

Reply:
Yes, this is what I meant. But I didn't want to mess up the code ^^